course_discovery/apps/api/v1/tests/test_views/test_course_runs.py

```diff
@@ -135,6 +135,22 @@ def test_list(self):
             self.serialize_course_run(CourseRun.objects.all().order_by(Lower('key')), many=True)
         )
 
+    def test_list_organization_filter(self):
+        """ Verify the endpoint returns a filtered list of course runs based on edly sub organization. """
+        edly_sub_org = 'edx'
+        query = 'organization=' + edly_sub_org
```

Comment:
Use verbose name for variables instead of generic name like `query`, `url`, `data`, `qs`

Reply:
Done